backend/plonk/bn254/solidity.go

```diff
@@ -422,7 +421,6 @@ contract PlonkVerifier {
   }
 
   function compute_pi(
-        bytes memory proof,
```

Comment:
I don't really understand dropping `proof` parameter from `compute_pi()`, since `load_wire_commitments_commit_api()` accepts the proof and uses it.  I guess you're planning further changes?

Reply:
Ah nice catch, the last test that I generated was without the use of api.Commit() and the proof is not needed in that case I forgot about that (I followed the compiler that told me the parameter was unused). I'll add a condition in the template to not take the proof when commit is not used, but it's a mistake on my end